rpc/http.go

```diff
@@ -65,21 +65,14 @@ func (hc *httpConn) Close() error {
 
 // DialHTTP creates a new RPC clients that connection to an RPC server over HTTP.
 func DialHTTP(endpoint string) (*Client, error) {
-	req, err := http.NewRequest("POST", endpoint, nil)
-	if err != nil {
-		return nil, err
-	}
-	req.Header.Set("Content-Type", "application/json")
-	req.Header.Set("Accept", "application/json")
-
 	initctx := context.Background()
 	return newClient(initctx, func(context.Context) (net.Conn, error) {
 		// add cookie support
 		cookieJar, _ := cookiejar.New(nil)
```

Comment:
why ignoring the error?

Reply:
nothing bad except out of memory can happen here